tests/test_digital.py

```diff
@@ -322,3 +323,38 @@ def test_dm(n_qubits: Tensor, batch_size: Tensor) -> None:
     dm = density_mat(state_cat)
     assert dm.size() == torch.Size([2**n_qubits, 2**n_qubits, batch_size])
     assert torch.allclose(dm, dm_proj)
+
+
+size = (5, 2)
+random_param = torch.randperm(size[0] * size[1])
+random_param = random_param.reshape(size)
+random_param = torch.sort(random_param, dim=1)[0]
+
+
+@pytest.mark.parametrize("target,n_qubits", random_param)
+@pytest.mark.parametrize("operator", [I, X, Y, Z, H, T, S])
```

Comment:
I would create a module-wide global variable since it is reused in two tests:

```python
GATESET = [I, X, Y, Z, H, T, S]
```

Reply:
Changed.